framework/mesh/mesh_continuum/mesh_continuum.h

```diff
@@ -186,6 +186,9 @@ class MeshContinuum
 
   void SetOrthoAttributes(const OrthoMeshAttributes& attrs) { ortho_attributes_ = attrs; }
 
+  /// Compute volume per material id's
+  void ComputeVolumePerMaterialID() const;
```

Comment:
```suggestion
  void ComputeVolumePerMaterialID() const;
```
Redundant comment. When the volumetric source conversion is finished we should remove `ID` from this.

Reply:
I do not know what to do. I orginally had `/** ... */` because other declarations just above were like that, then David asked me to go with `/// ...` and now you ask me to remove it altogether.

Reply:
David was correct that one-line comments should be `///` per our coding standards. My comment is that the documentation is redundant because it just restates the function name and that it is not necessary. @andrsd can weigh in on his thoughts about comments like this.

Reply:
+1 for me on that. If the function of the method is obvious from its name, I don't think it needs a comment.

Reply:
I prefer to add comments even if they seem like repeating the obvious in the function name for one and only one reason: The comment will show up in the doxygen page and when looked at by user they will see some text rather then nothing. Which can seem like developers did not care enough to document API, or they can feel it is experimental API they should not be using. Potentially also search may find the words, users are looking for...

Reply:
I suppose that is true. That is a lot of redundant comments throughout the codebase though... I think that commenting the function name with spaces is as documented as no comment to be honest. I don't know how the doxygen search feature is built but I don't know if it is built like Google where I could just search something like "computing volumes per material ID" and get a result. I have always searched based off of class name or function name.

Reply:
I see value in having this referenced by doxygen. For all one-liners, I am converting this header to use `/// ...` I hope this is ok with everyone.